julia_src/xpress/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
+# For creating base image of Julia API. Requires an untracked tar.gz file for FICO Xpress
+# Used in both production and local development
+
+WORKDIR /opt/reopt/solver
+COPY xp8.12.3_linux_x86_64.tar.gz .
+COPY install.sh .
```

Comment:
Maybe I'm not understanding what this is doing, but how can install.sh be copied if it's been deleted?

Reply:
This Dockerfile only has to built once. Then resulting image was pushed to Docker Hub and now our team can just pull it down. I have kept this Dockerfile in the repo as a record of how the base image was built.